.travis.yml

```diff
@@ -117,7 +117,7 @@ install:
   - pip install git+https://github.com/ansible/ansible.git@devel#egg=ansible
   - pip install git+https://github.com/sivel/ansible-testing.git#egg=ansible_testing
 script:
-  - python2.4 -m compileall -fq -x 'cloud/|monitoring/zabbix.*\.py|/dnf\.py|/layman\.py|/maven_artifact\.py|clustering/(consul.*|znode)\.py|notification/pushbullet\.py|database/influxdb/influxdb.*\.py|database/mssql/mssql_db\.py|/letsencrypt\.py|network/f5/bigip.*\.py|bmc/ipmi/.*\.py' .
+  - python2.4 -m compileall -fq -x 'cloud/|monitoring/zabbix.*\.py|/dnf\.py|/layman\.py|/maven_artifact\.py|clustering/(consul.*|znode)\.py|notification/pushbullet\.py|database/influxdb/influxdb.*\.py|database/mssql/mssql_db\.py|/letsencrypt\.py|network/f5/bigip.*\.py|bmc/ipmi/.*\.py|system/atomic_.*\.py' .
```

Comment:
>Does atomic run on RHEL5? If it doesn't maybe we can say it only works on python-2.6 and above because atomic is needed and atomic isn't available on the platforms where python-2.4 is the minimum version.

Since `atomic` interacts with Docker mostly (and `rpm-ostree` on Atomic Host), the support for `atomic` is basically in line with Docker.  In the case of RHEL releases, Docker is currently only supported for RHEL 7 [1].  And the same goes for `rpm-ostree`.

Additionally, RHEL7 shipped with python 2.7, so I believe `atomic` would need that version as a minimum.  Not sure what that means for this module, but just wanted to report this info.

[1] https://access.redhat.com/solutions/1378023